_includes/scripts.html

```diff
@@ -12,7 +12,7 @@
 <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js"></script>
 
 <!-- Compiled and minified JavaScript -->
-<script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
if we're going to optimize things, can we then also request jquery from the cloudflare cdn instead of code.jquery so we can reduce the DNS lookups? 

```https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js```

Reply:
Sure, that's fine. Updated.